css-view-transitions-1/Overview.bs

```diff
@@ -240,15 +270,17 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 		}
 		```
 
-		Note: This pseudo-element provides a containing block for all ''::view-transition-group()'' pseudo-elements.
-		The aim of the style is to size the pseudo-element to cover the [=snapshot viewport=]
-		and position all ''::view-transition-group()'' pseudo-elements relative to the [=snapshot viewport origin=].
+		<div class="note">
+			This pseudo-element provides a containing block for all ''::view-transition-group()'' pseudo-elements.
+			The aim of the style is to size the pseudo-element to cover the [=snapshot viewport=]
+			and position all ''::view-transition-group()'' pseudo-elements relative to the [=snapshot viewport origin=].
+		</div>
 
 	: <dfn>::view-transition-group( <<pt-name-selector>> )</dfn>
-	:: One of these pseudo-elements exists for each 'view-transition-name' in a view transition,
-		and holds the rest of the pseudo-elements corresponding to this 'view-transition-name'.
+	:: A [=tree-abiding pseudo-element=] that is also
+		a [=pseudo-element parent=], and a [=named view-transition pseudo-element=].
 
-		Its [=originating element=] is the ''::view-transition'' pseudo-element.
+		It is selected from its [=ultimate originating element=], the [=document element=].
```

Comment:
With this change, ie. without defining `::view-transition` as the originating pseudo-element of `::view-transition-group()`, it is not clear whether the latter is still a valid sub-pseudo-element of the former, ie. whether `::view-transition::view-transition-group(name)` is still valid, even if I do not think there are usefull use cases for such explicit selection (unlike with `::marker`, which selects `::before::marker` and `::after::marker`).

Maybe the intention is precisely to make it invalid?

For CSS parser/preprocessor authors, it makes a difference since invalid sub-pseudo-elements makes the selector invalid. I do not need to be explicitly defined, but I just would like to get a confirmation.

Reply:
This definition also appears in this PR:

> When a [=pseudo-element=] |pseudo| is part of a [=pseudo-element parent=] |parent|'s [=pseudo-element parent/children=], then |pseudo|'s [=originating pseudo-element=] is |parent|.

But, it's intended that `::view-transition::view-transition-group(name)` is invalid. We might do something with more tree-like selectors later.

Reply:
I came accross this note in CSS Lists and Counters: *an originating element that is a pseudo-element needs to be explicitly specified in the selector*. So the examples with `html::view-transition-*()` would be incorrect? `::before::marker` would be selected with `::marker` if this were not required, right?

Reply:
We explicitly resolved on a syntax which allows selecting these pseudo-elements directly from html, for example `html::view-transition-group(foo)` here: https://github.com/w3c/csswg-drafts/issues/7788#issuecomment-1292336905.

Reply:
Ok, I got it, sorry. Assuming `:div()` exists, `.foo:div(.bar)` would be equivalent to `.foo div.bar`, but there is no such equivalence with view transition pseudo-elements. The note in CSS List and Counters must not apply to functional pseudo-element.